jdaviz/configs/imviz/tests/test_delete_data.py

```diff
@@ -71,3 +74,34 @@ def test_delete_with_subset_wcs(self):
         assert_allclose(subset2.subset_state.roi.ymin, 0, atol=1e-6)
         assert_allclose(subset2.subset_state.roi.xmax, 3)
         assert_allclose(subset2.subset_state.roi.ymax, 2)
+
+
+class TestDeleteWCSLayerWithSubset(BaseImviz_WCS_GWCS):
```

Comment:
@rosteen , I tried to make a test for https://jira.stsci.edu/browse/JDAT-3958 but I cannot reproduce the failure! Could this bug only be triggered under certain condition? 

I also tried to reproduce the visual failure I saw earlier and I can no longer do it. Very perplexing. 🤯